unitree_legged_msgs/IMU imu
unitree_legged_msgs/MotorState[20] motorState
userValue_msg userValue
int32 userCmd